validator/src/main.rs

```diff
@@ -1867,6 +1867,22 @@ pub fn main() {
                 .hidden(true)
                 .help("Number of threads to use for servicing AccountsDb Replication requests"),
         )
+        .arg(
+            Arg::with_name("enable_accountsdb_plugin")
+                .long("enable-accountsdb-plugin")
+                .takes_value(false)
+                .requires("accountsdb_plugin_config")
+                .hidden(true)
+                .help("Enable AccountsDb plugin through which account data can streamed to external data store."),
+        )
```

Comment:
It has a little overlapping because of the "require". But I think this is a relatively significant feature for the validator and deserves a feature flag to control it explicitly to make the people more conscious about the risks/benefits of it.

Reply:
I think this complexity for the user is not worth it

Reply:
okay. removed. Thanks!